lensarg.js

```diff
@@ -32,12 +32,14 @@ module.exports = {
         console.log(error.noLens);
         process.exit(0);
       }
+
```

Comment:
I don't think it needs to get built into a "Lenses" subdirectory anymore... that was only useful when you could be generating zip file for multiple lenses from the ldk project... now the build is within the lens project itself, so I would say either choose to build the zip into a "dist" folder or just build it into the root. (I think "dist" might be better, maybe, just because it makes it explicit what it is for)

Reply:
But maybe you were planning on attacking that later...

Reply:
I'm actually not using lensarg.js in any script file and currently building the zip into the root. I didn't want to remove the files (like lensarg.js and webpack.config files) I'm not using from the repo.

Reply:
Maybe just create a new user story to capture all the "cleanup" stuff we want to get rid of after we're all done?

Reply:
I'll create one now as a reminder to deal with these